src/Module.Server/Common/BreakableWeaponsBehaviorServer.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Crpg.Module.Common.Network;
+using Crpg.Module.Notifications;
+using NetworkMessages.FromServer;
+using TaleWorlds.Core;
+using TaleWorlds.Engine;
+using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.MountAndBlade.Diamond;
+using TaleWorlds.PlayerServices;
+using Timer = TaleWorlds.Core.Timer;
+
+namespace Crpg.Module.Common;
+
+internal class BreakableWeaponsBehaviorServer : MissionBehavior
+{
+    public static readonly Dictionary<string, short> BreakAbleItemsHitPoints = new()
```

Comment:
Do you want to use this Dictionary forever for all Breakable items to come?
I would create an own Enum for that instead. Is more clean.